lib/mayaUsd/ufe/UsdUndoMaterialCommands.cpp

```diff
@@ -194,6 +196,16 @@ static const std::string kDefaultMaterialScopeName("mtl");
 
 void UsdUndoAssignNewMaterialCommand::execute()
 {
+    std::string materialsScopeName = kDefaultMaterialScopeName;
+    if (TfGetEnvSetting(USD_FORCE_DEFAULT_MATERIALS_SCOPE_NAME)) {
+        materialsScopeName = UsdUtilsGetMaterialsScopeName().GetString();
+    } else {
+        const char* mayaUsdDefaultMaterialsScopeName = std::getenv("MAYAUSD_MATERIALS_SCOPE_NAME");
+        if (mayaUsdDefaultMaterialsScopeName) {
+            materialsScopeName = std::string(mayaUsdDefaultMaterialsScopeName);
+        }
```

Comment:
Why not use "TfGetenv" which returns you a std::string and you can provide a default value if the env var is not set.